test/nools/task-emitter.spec.js

```diff
@@ -135,6 +335,36 @@ describe('task-emitter', () => {
           { _type:'_complete', _id: true },
         ]);
       });
+
+      it('should emit multiple tasks for multiple events', () => {
+        const task = aReportBasedTask();
+        task.events = [
+          { id: 'event1', days: 0, start: 0, end: 1 },
+          { id: 'event2', days: 1, start: 1, end: 2 },
+        ];
+        const config = {
+          c: personWithReports(aReport()),
+          targets: [],
+          tasks: [task],
+        };
+        const { emitted } = runNoolsLib(config);
+        expect(emitted.filter((e) => e._type === 'task')).to.have.length(2);
+        expectAllToHaveUniqueIds(emitted.filter((e) => e._type === 'task'));
```

Comment:
Can you add a priority calculation for multiple events here?

Reply:
The two events don't need to have the same score. The priority function takes the event as the parameter and the two events can end up having different scores. 

So you can have something like: 
```
task.priority = (c, r, e, d) => {
  if (e.id === 'event1') {
    return { level: 10 };
  }

  if (e.id === 'event2') {
    return { level: 20 };
  }
};
```

And then you would assert that the two generated tasks have different priorities.